src/hooks/index.ts

```diff
@@ -4,5 +4,5 @@ import { LDContext } from "../context/index"
 
 export const useFlags = () => {
   const data = useContext(LDContext)
-  return data?.flags || {}
+  return data ? data.flags : {}
```

Comment:
this is not exactly the same as before. it would return null if `data.flags` is null, whereas before it returned an empty object